helpers.md

```diff
@@ -585,6 +587,20 @@ If the method accepts route parameters, you may pass them as the second argument
 
     $url = action('UserController@profile', ['id' => 1]);
 
+<a name="method-asset"></a>
+#### `asset()` {#collection-method}
+
+Generate a URL for an asset.
+
+	$url = asset('img/photo.jpg');
+
+<a name="method-secure-asset"></a>
+#### `secure_asset()` {#collection-method}
+
+Generate a URL for an asset using HTTPS.
```

Comment:
It should be pointed out that `asset` will also generate an https link too if the current request scheme was https.